usng.js

```diff
@@ -265,6 +310,19 @@
             return result;
         },
 
+        serializeUTM({zoneNumber, easting, northing}) {
+            return `${zoneNumber} ${easting}mE ${northing}mN` 
```

Comment:
If we choose to use template strings and we don't compile them to es5, we are breaking compatibility with some older browsers we might want to support. Most projects that consume npm packages, don't include their dependencies in their own babel/webpack transformations. This is true of all other es6/7 features you wish to use.